kernel/tests/s11_rackscale_benchmarks.rs

```diff
@@ -130,23 +131,21 @@ fn rackscale_fxmark_benchmark(transport: RackscaleTransport) {
         )
     }
     fn timeout_fn(num_cores: usize) -> u64 {
-        120_000 + 20000 * num_cores as u64
+        180_000 + 5_000 * num_cores as u64
     }
     fn mem_fn(num_cores: usize, is_smoke: bool) -> usize {
         if is_smoke {
             8192
         } else {
-            core::cmp::max(73728, num_cores * 2048)
+            2048 * (((num_cores + 3 - 1) / 3) * 3)
```

Comment:
just curiosity: what's the logic behind the formula?

Reply:
It rounds up to the nearest number of cores that divisible by 3. Since we test with num replicas/worker 1, 2, 3, and 4, it needs to be divisible by all those numbers. For huge pages, I also got some errors that I believe were because you are supposed to have a multiple of huge pages.

Reply:
I can add a comment for this.